homeassistant/components/melissa/climate.py

```diff
@@ -61,12 +61,15 @@ class MelissaClimate(ClimateEntity):
     )
     _attr_temperature_unit = UnitOfTemperature.CELSIUS
 
-    def __init__(self, api, serial_number, init_data):
+    def __init__(self, api, device):
```

Comment:
This refactoring should be done in a preliminary PR. It distracts from the config-flow amends.